e2e/containers.go

```diff
@@ -95,71 +98,52 @@ func createHTTPServerContainer(alias string, tmpDir *helpertest.TmpFolder,
 	})
 }
 
-func createRedisContainer() (testcontainers.Container, error) {
-	ctx := context.Background()
-
-	req := testcontainers.ContainerRequest{
-		Image:          redisImage,
-		Networks:       []string{NetworkName},
-		ExposedPorts:   []string{"6379/tcp"},
-		NetworkAliases: map[string][]string{NetworkName: {"redis"}},
-		WaitingFor:     wait.ForExposedPort(),
+func WithNetwork(network string) testcontainers.CustomizeRequestOption {
```

Comment:
FYI we have just added a `WithNetwork` option here: https://github.com/testcontainers/testcontainers-go/pull/1894

Reply:
Thanks, this is very cool, I'll change our implementation 🤝